staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testing/webhook_server.go

```diff
@@ -49,7 +49,7 @@ func NewTestServer(t *testing.T) *httptest.Server {
 }
 
 func webhookHandler(w http.ResponseWriter, r *http.Request) {
-	fmt.Printf("got req: %v\n", r.URL.Path)
+	// fmt.Printf("got req: %v\n", r.URL.Path)
```

Comment:
intentional?

Reply:
yes, logging all requests in non-exceptional cases messed up use for benchmarks. left commented to make it easy to re-enable if debugging a test for some reason